docs/CONFIG-VARS.md

```diff
@@ -253,7 +254,8 @@ Additional node pools can be created separately from the default node pool. This
 
 | <div style="width:50px">Name</div> | <div style="width:150px">Description</div> | <div style="width:50px">Type</div> | <div style="width:75px">Default</div> | <div style="width:150px">Notes</div> |
 | :--- | :--- | :--- | :--- | :--- |
-| storage_type | Type of Storage. Valid Values: "standard", "ha"  | string | "standard" | A value of "standard" creates NFS server VM; a value of "ha" creates an AWS EFS mountpoint. |
+| storage_type | Type of Storage. Valid Values: "standard", "ha"  | string | "standard" | A value of "standard" creates a NFS server VM; a value of "ha" creates an AWS EFS mountpoint by default. |
+| storage_type_backend | The storage backend employed for the chosen `storage_type`. | string | "nfs" if "storage_type=standard"; "efs" if "storage_type=ha" | Valid Values: "nfs" if "storage_type=standard"; "efs" or "ontap" if "storage_type=ha" |
```

Comment:
The format here is quite difficult to read given the scenarios. Might want to simplify the presentation.